spec/enhancements/label_enhancement_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+RSpec.describe LabelEnhancement do
+  subject { described_class.new(property) }
+  let(:property) { SolrProperty.new("lcsubject_ssim", value) }
+  let(:value) { "http://localhost:40/1" }
+
+  describe "#property" do
+    context "when there's a label" do
+      it "should return a nice property" do
+        build_resource(uri: value, label: "Test")
```

Comment:
This is incorrect syntax - please update to `build_resource(uri: value, label: "Banana")`